.ci/pipeline_definitions

```diff
@@ -48,6 +48,7 @@ etcd-druid:
           nextversion: 'bump_minor'
           git_tags:
           - ref_template: 'refs/tags/{VERSION}'
+          release_callback: $REPO_DIR/.ci/bump_tag
```

Comment:
Where is `$REPO_DIR` being set?

Reply:
See https://gardener.github.io/cc-utils/traits/release.html --> `release_callback`

> an optional callback that is called during release commit creation. The callback is passed the absolute path to the main repository’s work tree via environment variable REPO_DIR. Any changes left inside the worktree are added to the resulting release commit